wacruit/src/apps/recruiting/schemas.py

```diff
@@ -0,0 +1,52 @@
+from datetime import datetime
+from typing import TYPE_CHECKING
+
+from wacruit.src.apps.common.enums import CodeSubmissionStatus
+from wacruit.src.apps.common.schemas import OrmModel
+
+if TYPE_CHECKING:
+    from wacruit.src.apps.problem.models import Problem
+
+
+class RecruitingListResponse(OrmModel):
+    id: int
+    name: str
+    is_active: bool
+    from_date: datetime
+    to_date: datetime
+    applicant_count: int
+
+
+class ProblemListResponse(OrmModel):
+    num: int
+    status: int
+
+    @classmethod
+    def from_orm(cls, problem: "Problem") -> "ProblemListResponse":
+        status = 0
+        if problem.submissions:
+            try:
+                next(
+                    filter(
+                        lambda submission: submission.status
+                        == CodeSubmissionStatus.SOLVED,
+                        problem.submissions,
+                    )
+                )
+                status = CodeSubmissionStatus.SOLVED.value
+            except StopIteration:
+                if problem.submissions[0].status == CodeSubmissionStatus.RUNNING:
+                    status = CodeSubmissionStatus.RUNNING.value
+                else:
+                    status = CodeSubmissionStatus.WRONG.value
+
+        return ProblemListResponse(num=problem.num, status=status)
+
+
+class RecruitingResponse(OrmModel):
```

Comment:
```suggestion
class MyRecruitingProblemsDto(OrmModel):
```
꼭 이거 아니라도 용도에 맞게 수정하면 좋을듯!